src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,16 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.length === 0
+      ? (<b data-cy="NoCommentsMessage">No comments yet</b>)
```

Comment:
better to render this tag outside of the `<div className="CommentList">`

Reply:
isn't fixed. Your CommentList should just get comments and render it. Make your component simple and move this condition to upper scope